allensdk/brain_observatory/ecephys/nwb/AIBS_ecephys_namespace.yaml

```diff
@@ -1,7 +1,9 @@
 namespaces:
-- doc: ""
+- doc: "Allen Institute ECEPHYS Extension"
   version: 0.2.0
   name: AIBS_ecephys
+  author: Allen Institute for Brain Science
```

Comment:
Does this need to be encapsulated in quotes (`"`) like the `doc`? Or are the quotes in the `doc` field unnecessary?

Reply:
No, in yaml you only need quotes to 'force a value to a string' if your value does not contain special characters yaml standard is to not use quotes.